go-controller/pkg/ovn/egressgw.go

```diff
@@ -106,6 +106,9 @@ func (oc *Controller) addGWRoutesForNamespace(namespace string, gws []net.IP, ns
 		gr := "GR_" + pod.Spec.NodeName
 		for _, gw := range gws {
 			for _, podIP := range pod.Status.PodIPs {
+				if (utilnet.IsIPv4(gw) && utilnet.IsIPv6(net.ParseIP(podIP.IP))) || (utilnet.IsIPv6(gw) && utilnet.IsIPv4(net.ParseIP(podIP.IP))) {
```

Comment:
I think that you can simplify it

`if utilnet.IsIPv6(gw) != utilnet.IsIPv6String(PodIP.IP) `

Reply:
I prefer more readable even if it's more verbose, but yeah, I come from Python, I don't have a strong opinion, will change.

Reply:
Yeah, what he said. Likewise below. (Also, huh, I hadn't seen that they added `IsIPv4` now too... there used to only be `IsIPv6`.)